### PR TITLE
feat(kudoctl): Update workload api usage

### PR DESCRIPTION
**Pull request by nponsard**
According to doc : https://thunder-random-3c6.notion.site/Http-API-8f21df70b6bc452486799f765e563fd0

### CODE REVIEW

File: kudoctl/src/client/workload.rs [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 
 use super::request::{Client, RequestError};
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct PortBinding {
+    pub source: i32,
+    pub destination: i32,
+}
+#[derive(Debug, Deserialize, Serialize)]
+pub struct WorkloadDTO {
```
Comment (sameo) [inline kudoctl/src/client/workload.rs:22]: What is a `WorkloadDTO` ?

Comment (nponsard) [reply kudoctl/src/client/workload.rs]: It’s the workload as stored in the controller, I added a comment in the code.

**Comment by sameo** [comment]
Please rebase against the `project-demo-202208` branch.

**Comment by nponsard** [comment]
changed the base to the dev branch

**Pull request merged**